plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceArea.java

```diff
@@ -160,19 +161,22 @@ private boolean validateSdk(Path location) {
     } catch (AppEngineException ex) {
       // accept a seemingly invalid location in case the SDK organization
       // has changed and the CloudSdk#validate() code is out of date
+      // todo: does this really make sense? In that case, wouldn't all the
```

Comment:
I took the todo out of this PR, though I'm still not convinced this shouldn't be a hard error. It comes down to probabilities. What's more likely? That the user has not installed the app engine Java components or that the layout of the Cloud SDK has changed in a way that throws th exception but doesn't break our functionality?